book/tutorials/cloud-computing/04-cloud-optimized-icesat2.ipynb

```diff
@@ -0,0 +1,147 @@
+    ":::\n",
+    "\n",
+    ":::{warning}\n",
+    "Generating cloud-native formats is non-trivial.\n",
+    ":::\n",
+    "\n",
+    ":::{seealso}\n",
+    "* https://eo-college.org/topics/cloud-native-data-formats/\n",
+    "* https://guide.cloudnativegeo.org\n",
+    ":::\n",
+    "\n",
+    "## Geoparquet\n",
+    "\n",
+    "To demonstrate one such cloud-native format, geoparquet, we have generated a geoparquet store (see [atl08_parquet.ipynb](./atl08_parquet_files/atl08_parquet.ipynb)) for the ATL08 dataset and will visualize it using a very performant geospatial vector visualization library, [`lonboard`](https://developmentseed.org/lonboard/latest/).\n",
+    "\n",
+    ":::{seealso} Resource on Geoparquet\n",
+    "* https://guide.cloudnativegeo.org/geoparquet/\n",
+    "* https://geoparquet.org/\n",
+    ":::\n",
+    "\n",
+    "## Demo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "editable": true,
+    "metadata": {
+     "mystnb": {
+      "skip-execution": true
+     }
+    },
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "import pyarrow.parquet as pq\n",
+    "from pyarrow import fs\n",
+    "import pyarrow.dataset as ds\n",
+    "from shapely import wkb\n",
+    "\n",
+    "s3  = fs.S3FileSystem(region=\"us-west-2\", anonymous=True)\n",
+    "dataset = pq.ParquetDataset(\"eodc-public/atl08_parquet/\", filesystem=s3,\n",
+    "                            partitioning=\"hive\", filters=[('year', '>=', 2021), ('year', '<=', 2021), ('month', '>=', 11), ('month', '<=', 11)])\n",
+    "table = dataset.read(columns=[\"h_canopy\", \"geometry\"])\n",
+    "df = table.to_pandas()\n",
+    "df['geometry'] = df['geometry'].apply(wkb.loads)\n",
+    "\n",
+    "\n",
+    "gdf = gpd.GeoDataFrame(df, geometry='geometry')\n",
+    "null_value = gdf['h_canopy'].max() # can we change this to a no data value?\n",
+    "gdf_filtered = gdf.loc[gdf['h_canopy'] != null_value]\n",
+    "gdf_filtered"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "gdf_filtered['h_canopy'].hist()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "from lonboard import Map, ScatterplotLayer\n",
+    "from lonboard.colormap import apply_continuous_cmap\n",
+    "from palettable.colorbrewer.diverging import BrBG_10\n",
+    "\n",
+    "min_bound = 0\n",
+    "max_bound = 60\n",
+    "h_canopy = gdf_filtered['h_canopy']\n",
+    "h_canopy_normalized = (h_canopy - min_bound) / (max_bound - min_bound)\n",
+    "\n",
+    "layer = ScatterplotLayer.from_geopandas(gdf_filtered, radius_min_pixels=0.5)\n",
```

Comment:
optional: note on how radius_min_pixels works (I noticed that zooming in on the map points get very small)